dist/common/scripts/scylla_util.py

```diff
@@ -87,13 +87,14 @@ def datadir():
 def scyllabindir():
     return str(scyllabindir_p())
 
+
 # @param headers dict of k:v
-def curl(url, byte=False, headers={}):
+def curl(url, headers=None, byte=False):
```

Comment:
Why not write headers={}?

The subject of the commit is wrong, since we could pass headers before. As fat as I can tell, this commit does nothing.

Reply:
I will fix the commit message. It was originally done before Lubus changes that added the headers argument.

Reply:
Thanks. This way I understand the change, and it can be backported independently if needed.

Reply:
Avi, FYI doing `headers={}` in a function signature, can bite someone very hard, 
since this dict is created only once. it basically becomes a global variable.

Reply:
Yes, unexpected for a nice language such as Python.